packages/terra-core-docs/src/terra-dev-site/doc/image/example/DecorativeImage/DecorativeImageVaraints.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { DecorativeImage } from 'terra-image';
+import decorativeImage from '../../assets/crocus.jpg';
+import placeholderPic1000x200 from '../../assets/1000x200.png';
+
+const DecorativeImageVaraints = () => (
+  <div>
+    <h2>Default</h2>
+    <DecorativeImage src={decorativeImage} />
+    <br />
+    <h2>Circle</h2>
+    <DecorativeImage src={decorativeImage} variant="circle" />
+    <br />
+    <h2>Rounded</h2>
+    <DecorativeImage src={decorativeImage} variant="rounded" />
+    <br />
+    <h2>Thumbnail</h2>
+    <DecorativeImage src={decorativeImage} variant="thumbnail" />
+    <br />
+    <h2>Fluid</h2>
+    <DecorativeImage src={decorativeImage} isFluid />
+    <br />
+    <h2>Placeholder</h2>
+    <h3>Invalid src prop (placeholder will show)</h3>
+    {/* eslint-disable-next-line react/jsx-no-duplicate-props */}
```

Comment:
Necessary?

Reply:
not required [removed](https://github.com/cerner/terra-core/pull/3585/commits/a72252e81ed3e48fbebc3ff03878cf6f428b0261#diff-bd0e9eec7cf28f31639dd12deaeb4b655703211779967016bd20c7a3274ed2cc) !!